Cannot find @Generated annotation for ServicesLogger

### Before reporting an issue

- [X] I have searched existing issues
- [X] I have reproduced the issue with the latest release

### Area

dependencies

### Describe the bug

When the `ServicesLogger` is generated via the JBoss logging processor, it's not possible to find `javax.annotation.Generated` annotation.

```
Failed to execute goal org.apache.maven.plugins:maven-javadoc-plugin:3.1.1:javadoc (aggregate-javadoc) on project keycloak-api-docs-dist: An error has occurred in Javadoc report generation: 
52977Error:  Exit code: 1 - /home/runner/work/keycloak-rel/keycloak-rel/services/target/generated-sources/annotations/org/keycloak/services/ServicesLogger_$logger.java:5: error: cannot find symbol
52978Error:  import javax.annotation.Generated;
52979Error:                         ^
52980Error:    symbol:   class Generated
52981Error:    location: package javax.annotation
52982Error:  /home/runner/work/keycloak-rel/keycloak-rel/services/target/generated-sources/annotations/org/keycloak/services/ServicesLogger_$logger.java:32: error: cannot find symbol
52983Error:  @Generated(value = "org.jboss.logging.processor.generator.model.MessageLoggerImplementor", date = "2023-04-28T01:09:06+0000")
52984Error:   ^
52985Error:    symbol: class Generated
```

### Version

999

### Expected behavior

No error

### Actual behavior

error: cannot find symbol

### How to Reproduce?

Build `distribution/api-docs-dist` module

### Anything else?

_No response_

--- a/services/pom.xml
+++ b/services/pom.xml
@@ -140,6 +140,10 @@
             <artifactId>jackson-annotations</artifactId>
         </dependency>
         <dependency>
+            <groupId>javax.annotation</groupId>
+            <artifactId>javax.annotation-api</artifactId>
+        </dependency>
+        <dependency>
             <groupId>com.fasterxml.jackson.datatype</groupId>
             <artifactId>jackson-datatype-jdk8</artifactId>
         </dependency>

